git-branchless-opts/src/lib.rs

```diff
@@ -320,6 +320,10 @@ pub struct RecordArgs {
     /// if any.
     #[clap(action, short = 'I', long = "insert")]
     pub insert: bool,
+
+    /// Detach the current branch before committing and then reattach.
```

Comment:
Since there may not be a branch explicitly involved, let's rewrite this, maybe something like

```suggestion
    /// After making the new commit, switch back to the previous commit.
```

Reply:
You are right. Done it.